store/tikv/lock_resolver.go

```diff
@@ -589,7 +596,7 @@ func (lr *LockResolver) resolvePessimisticLock(bo *Backoffer, l *Lock, cleanRegi
 		}
 		pessimisticRollbackReq := &kvrpcpb.PessimisticRollbackRequest{
 			StartVersion: l.TxnID,
-			ForUpdateTs:  math.MaxUint64,
+			ForUpdateTs:  l.LockForUpdateTS,
```

Comment:
in case tikv has not implement LockForUpdateTS, we need to fall back to MaxUint64.